docs/Installation/SSL.md

```diff
@@ -0,0 +1,204 @@
+
+1. Install Paralus
+2. Install LetsEncrypt Cert Manager - _you can use any other similar tool_
+3. Deploy ClusterIssuer & Certificate Objects
+4. Update annotations in HttpProxy
+
+
+Let’s discuss each of the steps in detail.
+
+## Steps to Enable SSL in Paralus
+
+### Install Paralus
+
+Based on the environment that you are using, you can refer to our [installation documentation](../installation) to install Paralus.
+
+### Install cert-manger
+
+cert-manager is a tool that adds certificates and certificate issuers as resource types in Kubernetes clusters, and simplifies the process of obtaining, renewing and using those certificates. Read more about [cert-manager](https://github.com/cert-manager/cert-manager).
+
+While it supports helm installation, we’ll use the yaml file provided
+
+```sh
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.5.4/cert-manager.yaml
+```
+
+This will install all the required components and CRDs in a new namespace. To validate the installation, run the following command:
+
+```sh
+$ kubectl -n cert-manager get all
+
+NAME                                       	     READY    STATUS	 RESTARTS  	AGE
+pod/cert-manager-6dd9658548-4k8br          	      1/1 	  Running   0          17s
+pod/cert-manager-cainjector-5987875fc7-jqxjj      1/1 	  Running   0          17s
+pod/cert-manager-webhook-7b4c5f579b-fcdmb  	      1/1 	  Running   0          17s
+
+NAME                       	      TYPE    	CLUSTER-IP   	    EXTERNAL-IP   PORT(S)	    AGE
+service/cert-manager       	    ClusterIP   10.245.254.139      <none>      9402/TCP    17s
+service/cert-manager-webhook    ClusterIP   10.245.207.16	      <none>      443/TCP	    17s
+
+NAME                                  	  READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/cert-manager          	  1/1 	      1        	1       	17s
+deployment.apps/cert-manager-cainjector   1/1 	      1        	1       	17s
+deployment.apps/cert-manager-webhook  	  1/1 	      1        	1       	17s
+
+NAME                                             	DESIRED   CURRENT   READY   AGE
+replicaset.apps/cert-manager-6dd9658548          	    1     	1     	  1    	17s
+replicaset.apps/cert-manager-cainjector-5987875fc7    1     	1     	  1   	17s
+replicaset.apps/cert-manager-webhook-7b4c5f579b  	    1     	1     	  1   	17s
+```
+
+### Deploy ClusterIssuer and Certificate Objects
+
+cert-manager supports two different CRDs for configuration:
+
+* `Issuer`, which is scoped to a single namespace
+* `ClusterIssuer` which is scoped to an entire cluster
+
+Since Paralus uses contour, for it to be able to serve HTTPS traffic for an Ingress in any namespace, use `ClusterIssuer`.
+
+> Note: Our default Paralus installation assumes that it is the only deployed application on a cluster and there are no other ingress, cert-issuers etc. present. In case you have multiple other ingresses present, you should use `Issuer` instead.
```

Comment:
```suggestion
> Note: Our default Paralus installation assumes that it is the only deployed application on a cluster and there are no other ingress, cert-issuers etc. present. In case you have multiple other ingresses present, you should use `Issuer` instead and deploy it on the same namespace as Paralus. refer: [Difference between Issuer and ClusterIssuer](https://cert-manager.io/docs/concepts/issuer/#namespaces)
```